src/imgui/ImGuiMachine.cc

```diff
@@ -31,6 +31,20 @@ using namespace std::literals;
 
 namespace openmsx {
 
+void ImGuiMachine::save(ImGuiTextBuffer& buf)
+{
+	for (const auto& item : recentMachines) {
+		buf.appendf("machine.name=%s\n", item.c_str());
```

Comment:
Detail: `name` is very generic. Maybe it's more descriptive to rename this to `recent`? (and then also in `loadLine()` of course).

Reply:
OK, changed it in the latest version.